### PR TITLE
Update Transparent Statement to Signed Statement. 

### PR DESCRIPTION
**Pull request by SteveLasker**
from [scitt alias feedback](https://mailarchive.ietf.org/arch/msg/scitt/O3nkTYsEqYzA-DF1wVW-Q7cpJn4/):
> In section 4.1.1.2 it alludes that an auditor can mine the log to authenticate and retrieve the transparent statement.  Based on the last statement this has two problems.  The first it assumes that the signed statement is kept and that it can generate a new receipt (see 3) and produce a transparent statement.  If it could this would be equivalent to the original in intent but not binary equal.   The original receipt (or last created receipt from a refresh) is not required (or intended) to be kept in the ledger.  Generically then this is impossible and may be where we would leverage consistency proofs in a later version.

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -410,7 +410,7 @@ The Transparency Service MUST apply the Registration Policy that was most recent
 
 The operator of a Transparency Service MAY update the Registration Policy or the trust anchors of a Transparency Service at any time.
 
-Transparency Services MUST ensure that for any Signed Statement they register, enough information is made available to Auditors (either in the Append-only Log and retrievable through audit APIs, or included in the Receipt) to authenticate and retrieve the Transparent Statements describing the registration policy and trust anchors that apply to this registration.
+Transparency Services MUST ensure that for any Signed Statement they register, enough information is made available to Auditors (either in the Append-only Log and retrievable through audit APIs, or included in the Receipt) to authenticate and retrieve the Signed Statements describing the registration policy and trust anchors that apply to this registration.
```
Comment (aj-stein-nist) [inline draft-ietf-scitt-architecture.md:413]: MUST and "enough" does not seem sufficient for a requirement.

**Review by aj-stein-nist — CHANGES_REQUESTED** [review]
MUST and "enough" does not seem suffi

**Review by henkbirkholz — APPROVED** [review]
more correct than before, still a MUST that is tad bit hard to follow. the change in this scope is okay
